sal_interfaces/MTAlignment/MTAlignment_Commands.xml

```diff
@@ -70,6 +123,17 @@
       <Count>1</Count>
     </item>
   </SALCommand>
+  <SALCommand>
+    <Subsystem>MTAlignment</Subsystem>
+    <EFDB_Topic>MTAlignment_command_powerOff</EFDB_Topic>
+    <item>
+      <EFDB_Name>powerOff</EFDB_Name>
+      <Description>turn off the tracker and controller, and closes the interface</Description>
```

Comment:
Please use an active voice. Also, the item name is `powerOff` so please use a phrase like "power off the tracker and controller, and close the interface".